PotentialGapPlannerROS:
    min_resoln: 1
    gap_diff: 0.1
    global_plan_lookup_increment: 30
    goal_tolerance: 0.5
    global_plan_change_tolerance: 0.1
    follow_the_gap_visparsing: True
    close_gap_visparsing: True

    epsilon2: 0.20
    epsilon1: 0.20
    r_inscr:  0.20

    niGen_s: False
    config_sanity_val: 100
    ctrl_ahead_pose: 2
    reduction_threshold: 1.5708
    max_idx_diff: 128
    feasi_inflated: True
    projection_inflated: False
    planning_inflated: False
    fig_gen: False
    far_feasible: False

    synthesized_frame: True
    scale: 1
    integrate_maxt: 10
    integrate_stept: 0.01

    num_feasi_check: 30